docs-src/cloud/users-delete.md

```diff
@@ -0,0 +1,38 @@
+---
+id: users-delete
+title: How to delete a user from your Temporal Cloud account
+sidebar_label: Delete users
+description: Use Settings in Web UI or the `tcld user delete` command.
+tags:
+  - temporal cloud
+  - how-to
+  - users
+  - temporal cloud account
+---
+
+You can delete a user from your Temporal Cloud Account by using either Web UI or tcld.
+
+:::info
+
+To delete a user, a user must have the Global Admin account-level [Role](/cloud/#account-level-roles).
+
+:::
+
+<!-- How to delete a user from your Temporal Cloud account using Web UI -->
+
+### How to update an account-level Role using Web UI
+
+1. In Temporal Web UI, select **Settings** in the left portion of the window.
+1. On the **Settings** page, find the user and, on the right end of the row, select **Delete**.
+1. In the **Delete User** dialog, select **Delete**.
+
+You can delete a user in two other ways in Web UI:
+
+- On the user profile page, select the down arrow next to **Edit User** and then select **Delete**.
+- On the **Edit User** page, select **Delete User**.
```

Comment:
In this case, this should be an unordered list because these are two alternative ways to accomplish the same task as the preceding procedure. However, I can see that it's easy to misread. I'll add "Or" to the beginning of the second list item to help distinguish it as a separate choice.

Having said that, I also considered leaving out this part entirely, given the simplicity of the preceding procedure. The alternatives are discoverable, so maybe the docs don't need to mention them. Anyone have opinions on that approach?

Reply:
I chose a somewhat different route. What do you think of this approach?

```
- User profile page: Select the down arrow next to **Edit User** and then select **Delete**.
- **Edit User** page: Select **Delete User**.
```